configReader.py

```diff
@@ -6,31 +6,29 @@
 """
 import yaml
 
-##to do
-#implement singleton pattern here
+
+# to do
+# implement singleton pattern here
 class ConfigurationReader():
-    __APIKEY=None
-    __WebsiteSupported=[]
-    __Limit=None
+    __APIKEY = None
+    __WebsiteSupported = []
+    __Limit = None
+
     def __init__(self):
         with open("config.yml", 'r') as ymlfile:
             cfg = yaml.load(ymlfile)
-        ConfigurationReader.__APIKEY=cfg['Apikey']
-        #print ConfigurationReader.__APIKEY
-        ConfigurationReader.__Limit=cfg['Limit']
-        #print ConfigurationReader.__Limit
-        ConfigurationReader.__WebsiteSupported=cfg['WebsiteSupported']
-        #print ConfigurationReader.__WebsiteSupported

```

Comment:
i dont know about the the pep8 . Can u let me know why the comment has been removed?